cdap-cli/src/main/java/co/cask/cdap/cli/util/AbstractAuthCommand.java

```diff
@@ -72,29 +76,38 @@ public void execute(Arguments arguments, PrintStream printStream) throws Excepti
   public abstract void perform(Arguments arguments, PrintStream printStream) throws Exception;
 
   protected ServiceId parseServiceId(Arguments arguments) {
```

Comment:
in fact, if you address my comment below, you shouldn't need this method in this class at all, since it will become a 1-liner method.